changes/139.changes

```diff
@@ -0,0 +1 @@
+Now gracefully handles browser page refreshes on `POST` views.
```

Comment:
I thought an F5 on a post request would pop up a window asking if the user wanted to re-send the values (i.e. send a new post request) Has this browser behaviour changed?

Reply:
It does, but what the flask view methods were doing on POST were returning a render template. And because the method gets a return, flask completes the request with the 200 OK. so the POST already completed successfully in the eyes of the browser. this behaviour was different in pylons which is why we had return renders in the POST view methods.